src/specs/api.v2.yaml

```diff
@@ -1154,7 +1186,56 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPError'
-
+  '/experiments/{experimentId}/clone/{toExperimentId}':
```

Comment:
I'm not clear where this `toExperimentId` would come. I think it should be the UI creating it

Reply:
Same as https://github.com/hms-dbmi-cellenics/api/pull/377#discussion_r904927722